newsfragments/3308.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Ensure that the application is build in release mode from the CI without too much hustle.
```

Comment:
No need for a newsfragment (on top of that we have always compiled in release for parsec-cloud ^^)

Reply:
done